public/index.html

```diff
@@ -136,10 +136,7 @@ <h2 class="bd-logo-subtitle text-white">Registra in Developers Italia il tuo str
 
         <div class="d-flex pt-2">
           <div class="px-2 team-logo--icon">
-            <img src="assets/logo-mid-white.svg" alt="Dipartimento per la Trasformazione Digitale" />
-          </div>
-          <div class="px-2 border-white border-left team-logo--text text-white">
```

Comment:
@bfabio should we also remove https://github.com/italia/developers-italia-onboarding/blob/23079447ceb02e56b3324aa892c5799ec271a0e9/public/assets/css/main.css#L55?

Reply:
@libremente nice catch